### PR TITLE
Avoid to throw an exception when we cannot update the status of the p…

### PR DESCRIPTION
**Pull request by v4lux**
…aypal account and create a waiting status

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Does this PR brings an improvement or a bug fix?

File: _dev/src/store/modules/paypal/actions.js [hunk 1]
```diff
@@ -43,4 +43,13 @@ export default {
       return Promise.resolve();
     });
   },
+  updatePaypalStatus({commit, getters}) {
```
Comment (Quetzacoalt91) [inline _dev/src/store/modules/paypal/actions.js]: By using `update`, I understand that you're trying to send data to store to a controller. `refreshPaypalStatus` seems more appropriate.

File: classes/Entity/PaypalAccount.php [hunk 2]
```diff
@@ -71,10 +71,6 @@ class PaypalAccount
 
     public function __construct($merchantId = null, $email = null, $emailIsVerified = null, $paypalPaymentStatus = null, $cardPaymentStatus = null)
     {
-        if (empty($merchantId)) {
-            throw new \PrestaShopException('merchantId cannot be empty');
-        }
```
Comment (Quetzacoalt91) [inline classes/Entity/PaypalAccount.php]: `$merchantId` is now an optional parameter?

File: _dev/src/App.vue [hunk 3]
```diff
@@ -72,6 +72,18 @@
...
+    data() {
+      return {
+        paypalStatusUpdater: null,
+      };
+    },
+    methods: {
+      updater() {
+        this.paypalStatusUpdater = setInterval(() => {
+          this.$store.dispatch('updatePaypalStatus');
+        }, 5000);
```
Comment (Quetzacoalt91) [inline _dev/src/App.vue]: This value seems low, I guess we can double it to 10s.

File: _dev/src/store/modules/paypal/actions.js (hunk 1 above)
Comment (v4lux) [reply _dev/src/store/modules/paypal/actions.js]: Renamed to refreshPaypalStatus() for a better understanding.

File: _dev/src/App.vue (hunk 3 above)
Comment (v4lux) [reply _dev/src/App.vue]: Changed to 10 seconds.

**Pull request merged**